src/validation.cpp

```diff
@@ -1084,6 +1084,22 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
         m_viewmempool.PackageAddTransaction(ws.m_ptx);
     }
 
+    // Apply package mempool ancestor/descendant limits.
+    std::string err_string;
+    if (!m_pool.CheckPackageLimits(txns, m_limit_ancestors, m_limit_ancestor_size, m_limit_descendants,
+                                   m_limit_descendant_size, err_string)) {
+        // All transactions must have individually passed mempool ancestor and descendant limits
+        // inside of PreChecks(). Figuring out which transaction to attribute this failure to may be
+        // implementation-dependent, and it's likely to be multiple transactions because we
+        // evaluated all of them together. Return the same failure for all transactions.
+        for (auto& ws : workspaces) {
+            ws.m_state.Invalid(TxValidationResult::TX_MEMPOOL_POLICY, "exceeds-ancestor-descendant-limits", err_string);
+            results.emplace(ws.m_ptx->GetWitnessHash(), MempoolAcceptResult::Failure(ws.m_state));
```

Comment:
That's a good point - perhaps it would be better to copy the error string from the `TxValidationState` into the `PackageValidationState` and always return results empty when we have a package-wide error.

Reply:
Thanks for the suggestion - I got rid of the tx results when there's a package-wide error, and added a comment for `m_tx_results` in validation.h